src/Mobs/Monster.cpp

```diff
@@ -183,7 +183,7 @@ bool cMonster::TickPathFinding(cChunk & a_Chunk)
 
 		case ePathFinderStatus::PATH_NOT_FOUND:
 		{
-			ResetPathFinding();  // Try to calculate a path again.
+			StopMovingToPosition();  // Try to calculate a path again.
```

Comment:
Reason for changing this line: Forgot :P
But it was very justified, I'll recall and write it.